src/scss/components/_cards.scss

```diff
@@ -12,8 +12,7 @@ $card-bgColor--disabled                         : $grey--lightest;
 $card-tooltip-width                             : 10px;
 $card-arrow-bottom-position                     : 0;
 $card-padding                                   : spacing(x2);
-$card-radius                                    : 4px;
-$card-radius--large                             : 8px;
+$card-radius                                    : spacing(x0.5);
```

Comment:
Not a big deal as it's unlikely to change atm, but not sure we'd want spacing linked to the border-radius like this. Would be weird if the spacing increased that corners of components also were affected.

Reply:
Yeah, good shout, I get a bit carried away with the spacing helper sometimes! I'll update this soon when I next put a PR in.

Reply:
This was done in #280.

Reply:
Ah no it wasn't, I updated the wrong thing!

Reply:
Fixed in #281 😸